modules/lib/lib-task/src/main/resources/lib/xp/task.ts

```diff
@@ -107,9 +107,9 @@ export function executeFunction(params: ExecuteFunctionParams): string {
     return bean.executeFunction();
 }
 
-export interface SubmitNamedTaskParams {
+export interface SubmitNamedTaskParams<Config extends object = never> {
```

Comment:
I think `never` as a default value is very good for TypeScript-developers, but it might trigger an error for JavaScript-developers using XP, since they may not be likely to set the type parameter.

So I suggest a more "flexible" default value:
```suggestion
export interface SubmitNamedTaskParams<Config extends object = Record<string, unknown>> {
```

Reply:
Updated